packages/component-library/src/index.ts

```diff
@@ -94,4 +96,6 @@ export {
   MtPagination as SwPagination,
   // @deprecated
   MtSkeletonBar as SwSkeletonBar,
+  // @deprecated
+  MtToast as SwToast,
```

Comment:
Making things backward compatible makes sense for the components we already offer. I suggest that we don't offer that for new components for the following reasons.
1. We don't offer stuff we directly deprecate
2. Leaving this out means that people have less to migrate because they instantly use the correct one